contracts/BeakerContract.sol

```diff
@@ -149,6 +149,62 @@ contract BeakerContract is IKernel {
         }
   }
 
+  function proc_acc_call(uint8 capIndex, address account, uint256 amount, bytes memory input) internal returns (uint32 err, bytes memory output) {
```

Comment:
Noticed you're opting out of including `fselector` like in `#proc_call`. I'm guessing in preparation for solc `^0.5.6` with `abi#encode` ?

Reply:
In fact it was only just yesterday when updating call procedure that I realised you'd included fselector, which is just a higher level abstraction. What are your thoughts? I had always imagined just passing a big bag of bytes. I think if we can get Solidity to do all that encoding for us it's a clear winner. If not having a separate fselector parameter makes sense.

Reply:
I think so too, it should be ok for now. I think.